great_expectations/data_context/data_context/base_data_context.py

```diff
@@ -252,6 +252,54 @@ class BaseDataContext(EphemeralDataContext, ConfigPeer):
     GE_YML = "great_expectations.yml"  # TODO: migrate this to FileDataContext. Still needed by DataContext
     GE_EDIT_NOTEBOOK_DIR = GE_UNCOMMITTED_DIR
     DOLLAR_SIGN_ESCAPE_STRING = r"\$"
+    TEST_YAML_CONFIG_SUPPORTED_STORE_TYPES = [
```

Comment:
@lccasagrande I was curious -- how come this very general code is here?  It this something you wrote?  If so, for what purpose?  Otherwise, is this the case of a bad merge?  Thank you!

Reply:
I did not write this. Do not know what it does.

Reply:
@lccasagrande Thank you.  I find it odd that it shows up as new code in your pull request.  I will ask my co-workers.

Reply:
Hi @lccasagrande! You can go ahead and remove the changes from this file and revert to what is in develop. It was moved into `abstract_data_context.py`

Reply:
🙇

Reply:
Done